eslint/eslint-config/profile/_common.js

```diff
@@ -260,54 +260,10 @@ function buildRules(profile) {
           '@typescript-eslint/adjacent-overload-signatures': 'warn',
 
           // STANDARDIZED BY:   @typescript-eslint\eslint-plugin\dist\configs\recommended.json
-          //
-          // CONFIGURATION:     By default, these are banned: String, Boolean, Number, Object, Symbol
-          '@typescript-eslint/ban-types': [
```

Comment:
Why are we removing this rule?

Reply:
It's been deprecated and replaced with more specific rules. See the deprecation note here: https://typescript-eslint.io/rules/ban-types/

Reply:
Have we added those rules?